### PR TITLE
Keyboard events handling

### PR DESCRIPTION
**Pull request by ala-n**
- Array.prototype.include polyfill
- Keycodes.prototype.key shim
- Deprecated which/keycode properties no longer supported

### CODE REVIEW

**Comment by ala-n** [comment]
And, basically, one open question for style-guide:
When should we prefer `Array.includes` condition instead of simple comparison: `[A, B].incluses(var)` vs `A === var || B === var`
Prefer `[A, B].incluses(var)` ...
 - never  
 - for 2 and more literals check
 - for 3 and more literals check
Leave your thoughts in the thread for that comment

**Review by YanaBr — APPROVED** [review]

**Review by andreybelous — APPROVED** [review]

**Review by julia-murashko — APPROVED** [review]

File: src/modules/esl-base-trigger/core/esl-base-trigger.ts [hunk 1]
```diff
@@ -110,12 +110,9 @@ export class ESLBaseTrigger extends ESLBaseElement {
 
   @bind
   protected _onKeydown(e: KeyboardEvent) {
-    switch (e.which || e.keyCode) {
-      case ENTER:
-      case SPACE:
-        this.click();
-        e.preventDefault();
-        break;
+    if ([ENTER, SPACE].includes(e.key)) {
```
Comment (dshovchko) [inline src/modules/esl-base-trigger/core/esl-base-trigger.ts:113]: :+1:

**Review by dshovchko — APPROVED** [review]

File: src/modules/esl-trigger/core/esl-triggers-container.ts [hunk 2]
```diff
@@ -34,17 +35,15 @@ export class ESLTriggersContainer extends ESLBaseElement {
...
-        this.goTo('next', target);
-        e.preventDefault();
-        break;
+    if ([ARROW_UP, ARROW_LEFT].includes(e.key)) {
+      this.goTo('prev', target);
+      e.preventDefault();
+    }
+
+    if ([ARROW_DOWN, ARROW_RIGHT].includes(e.key)) {
+      console.log('right');
```
Comment (yadamskaya) [inline src/modules/esl-trigger/core/esl-triggers-container.ts:44]: I guess you forgot to remove it, right?

**Review by yadamskaya — APPROVED** [review]

**Review by nattallius — APPROVED** [review]

**Pull request merged**